core/src/com/unciv/ui/options/GameplayTab.kt

```diff
@@ -18,6 +18,7 @@ fun gameplayTab(
     val settings = optionsPopup.settings
 
     optionsPopup.addCheckbox(this, "Check for idle units", settings.checkForDueUnits, true) { settings.checkForDueUnits = it }
+    optionsPopup.addCheckbox(this, "Auto Unit Cycle", settings.autoUnitCycle, true) { settings.autoUnitCycle = it }
```

Comment:
These two texts need to be added to template.properties so they will be translatable